mon-pix/app/components/progress-bar.js

```diff
@@ -1,41 +1,31 @@
-import { computed } from '@ember/object';
+import Component from '@glimmer/component';
 import { inject as service } from '@ember/service';
 import { htmlSafe } from '@ember/string';
 import colorGradient from 'mon-pix/utils/color-gradient';
-import Component from '@ember/component';
-import classic from 'ember-classic-decorator';
 
-const { and } = computed;
-
-const minWidthPercent = 1.7;
-const minWidthPixel = 16;
-
-@classic
 export default class ProgressBar extends Component {
   @service media;
   @service progressInAssessment;
 
-  @and('media.isDesktop', 'assessment.showProgressBar')
-  showProgressBar;
+  minWidthPercent = 1.7;
+  minWidthPixel = 16;
```

Comment:
Je propose de les remettre en dehors de la classe si possible et de les passer en CAPS 
```suggestion
  MINIMUM_WIDTH_STEP_IN_PERCENT = 1.7;
  MINIMUM_WIDTH_STEP_IN_PIXEL = 16;
```